src/common/time/src/timestamp.rs

```diff
@@ -381,7 +381,7 @@ impl Timestamp {
 
     pub fn from_chrono_datetime(ndt: NaiveDateTime) -> Option<Self> {
         let sec = ndt.and_utc().timestamp();
-        let nsec = ndt.timestamp_subsec_nanos();
+        let nsec = ndt.and_utc().timestamp_subsec_nanos();
```

Comment:
This seems to change the semantic. Why do we need it?

Reply:
it's deprecated